integration_tests/tests/apply/view_tasklist.cy.ts

```diff
@@ -48,6 +48,9 @@ context('New', () => {
     //  click on the start button
     cy.get('[role="button"]').click()
 
+    // click on the start a new application button
```

Comment:
think this will come out when you rebase against main, but we no longer test from the index page here.